src/bci_build/package.py

```diff
@@ -1106,8 +1106,7 @@ def _get_python_kwargs(
         "version": py3_ver,
         "additional_versions": ["3"],
         "env": {"PYTHON_VERSION": py3_ver_replacement, "PIP_VERSION": pip3_replacement},
-        "package_list": [py3, pip3, "curl", "git-core"]
-        + (
+        "package_list": [f"{py3}-devel", pip3, "curl", "git-core"] + (
```

Comment:
I am not convinced that this is really necessary. Only some packages require python3-devel to be build and this makes the container bigger.

Reply:
it makes the container bigger, but it also makes the container more stable because mismatches between repo and container (think about people pinning a container) no longer breaks by default